src/sentry/snuba/subscriptions.py

```diff
@@ -156,14 +167,20 @@ def create_snuba_subscription(
     :param snuba_query: A `SnubaQuery` instance to subscribe the project to.
     :return: The QuerySubscription representing the subscription
     """
+    timebox_start = None
+    if timebox:
+        timebox_start = timezone.now()
```

Comment:
Rather than passing in `timebox` could we just have the user pass in `timebox_start`? That would make this a bit more flexible allowing us to set the start to whatever we want to, not just `timezone.now()`

Reply:
Yea - either way could work.
figured it would make the most sense to grab the timestamp closest to creating our model instance though since we _don't_ actually want to make this more flexible.

I'm essentially restricting us from passing in any timestamps we want - and _only_ grabbing the current now timestamp